### PR TITLE
Updated info.json and operation.json

### PR DESCRIPTION
**Pull request by cs-sujeet-ahirrao**
- Updated action descriptions and output schema in info.json
- save the output file in env for auto deletion

### CODE REVIEW

File: cicd-utils/info.json [hunk 1]
```diff
@@ -29,8 +30,8 @@
           "visible": true,
           "editable": true,
           "placeholder": "Eg. 5af00c658289bb5618193f42",
-          "description": "Specify the file path on the filesystem",
-          "tooltip": "filepath under /tmp directory"
+          "description": "Specify the export template zip file path on the filesystem",
+          "tooltip": "Filepath under /tmp directory"
```
Comment (cs-suyog-jinturkar) [inline cicd-utils/info.json]: Suggestion: export template zip file path under /tmp directory

File: cicd-utils/info.json [hunk 2]
```diff
@@ -29,8 +30,8 @@
           "visible": true,
           "editable": true,
           "placeholder": "Eg. 5af00c658289bb5618193f42",
```
Comment (cs-suyog-jinturkar) [inline cicd-utils/info.json:32]: e.g. 5af00c658289bb5618193f42.zip

File: cicd-utils/info.json [hunk 3]
```diff
@@ -17,8 +17,9 @@
       "operation": "unzip_export_template",
       "annotation": "unzip_export_template",
       "category": "utilities",
-      "description": "Unzip the specified downloaded zip export template",
+      "description": "Unzip the specified downloaded export template zip file",
```
Comment (cs-suyog-jinturkar) [inline cicd-utils/info.json]: Suggestion: Unzip the specified export template zip

File: cicd-utils/info.json [hunk 4]
```diff
@@ -40,8 +41,9 @@
       "operation": "split_export_templates",
       "annotation": "split_export_templates",
       "category": "utilities",
-      "description": "Split the export template recordset into Production Content, Production Settings and Development Settings export template recordset",
+      "description": "Split the export template file into Production Content, Production Settings and Development Settings export template files",
```
Comment (cs-suyog-jinturkar) [inline cicd-utils/info.json]: Suggestion: Split the export template file into three export templates named Production Content, Production Settings and Development Settings

File: cicd-utils/playbooks/playbooks.json [hunk 5]
```diff
@@ -2,45 +2,45 @@
...
           "@type": "Workflow",
-          "uuid": "552990bb-c0cf-40fd-a3db-0f2410bb5d28",
-          "collection": "/api/3/workflow_collections/f902edbe-18c8-40f0-a231-27fed7bdf160",
+          "uuid": "f885a78a-63fc-40ae-b748-c550e2454d60",
+          "collection": "/api/3/workflow_collections/1d55aff3-d229-451a-b36c-13ecf76ccfbc",
           "triggerLimit": null,
-          "description": "Takes relative date (last x days, for example), time period (hourly, daily, etc), date format string, plus optionally a list of existing datetimes and builds a list of time buckets from them spanning the time window specified by the relative date information.",
-          "name": "Assemble Query Time Windows",
-          "tag": "#Time Series Chart Utilities",
+          "description": "Unzip the specified downloaded export template zip file",
```
Comment (cs-suyog-jinturkar) [inline cicd-utils/playbooks/playbooks.json]: Unzip the specified export template zip file

File: cicd-utils/info.json (hunk 2 above)
Comment (cs-sujeet-ahirrao) [reply cicd-utils/info.json:32]: in filepath .zip extension will not present so no need to add .zip extension

File: cicd-utils/info.json (hunk 1 above)
Comment (cs-sujeet-ahirrao) [reply cicd-utils/info.json]: done

File: cicd-utils/info.json (hunk 3 above)
Comment (cs-sujeet-ahirrao) [reply cicd-utils/info.json]: done

File: cicd-utils/info.json (hunk 4 above)
Comment (cs-sujeet-ahirrao) [reply cicd-utils/info.json]: done

**Review by cs-suyog-jinturkar — APPROVED** [review]

**Pull request merged**